CHARTER.adoc

```diff
@@ -46,15 +57,15 @@ The TODO Group will host Working Groups for members to contribute to that are in
  ... Meetings will occur at least once per quarter.
  ... Details of meeting mechanisms and instructions for attendance will be published reasonably in advance to all members in good standing.
  .. Communication mechanisms
- ... All public communication not during meetings will take place over the public group mailing list(s) accessible to members and non-members alike.
- ... All private communication not during meetings will take place over the private group mailing list for members only.
+ ... All public communication not during meetings will take place over the public group mailing list(s), real-time chats, issue trackers, or source code management systems and will be accessible to members and non-members alike.
```

Comment:
```suggestion
... Communication is divided between community (public) and general members (private). The general members at times share sensitive information and use a private channel for that type of discussion. Information shared in those channels is later anonymized and shared with the wider OSPO community via [OSPO Discussions](https://github.com/todogroup/ospology/discussions).

 ... All public communication not during meetings will take place over the public group mailing list(s), real-time chats, issue trackers, or source code management systems and will be accessible to members and non-members alike.
```

Reply:
This is fine for me to include in the charter.

Reply:
LGTM. Somewhat nit picky... Are all discussions from the private channel anonymized and shared in OSPO Discussions? If not, can say "Some of the information shared in those channels are later anonymized..."

Reply:
LGTM with @annania's recommendation to say "some of the information" as it is an accurate synopsis of our operating processes. Please merge after that change has been made.